include/swift/AST/DiagnosticsSema.def

```diff
@@ -2655,6 +2655,8 @@ WARNING(add_predates_concurrency_import,none,
      "%select{| as warnings}0", (bool, Identifier))
 WARNING(remove_predates_concurrency_import,none,
      "'@preconcurrency' attribute on module %0 has no effect", (Identifier))
+NOTE(add_preconcurrency_to_conformance,none,
+     "add '@preconcurrency' to the %0 conformance to suppress isolation-related diagnostics", (DeclName))
```

Comment:
Instead of just saying this will suppress diagnostics, should we say that applying `@preconcurrency` will switch from static isolation diagnostics to dynamic ones? I think this should somehow mention that you're opting into dynamic assertions by doing this.

Reply:
That is a *great* point! Thank you!